### DISCUSSION TITLE
Max fan speed calibration

### DISCUSSION THREAD

**Discussion by justinhow** [Rem0o/FanControl.Releases#1672] [General]
I have just come across this great fan control software. Just exactly what I was looking for. I think I understand most of it but I am not sure how to calibrate software so that 100% fan speed is calibrated to the physical fan itself. I see the stop and start % calibrations which is good but no max speed calibration. The reason I would want this is that in the graphs the fan is not maxing out when the graph is actually at just 76% for example.
Sorry if this is a stupid question or there is another place/forum more suitable for it.

**Comment by Rem0o** [comment]
Max speed is well, 100%? No need for calibration. These fans are designed to be at max speed at 12V, which is 100%.

**Reply by justinhow** [reply]
I was just about to answer myself as my testing had concluded that max speed does not need ot be calibrated. Makes sense.
Thanks
